libs/payments/stripe/src/lib/stripe.manager.spec.ts

```diff
@@ -370,34 +368,23 @@ describe('StripeManager', () => {
 
   describe('getPlan', () => {
     it('returns plan', async () => {
-      const mockPlan = StripePlanFactory();
+      const mockPlan = StripeResponseFactory(StripePlanFactory());
 
-      mockClient.plansRetrieve = jest.fn().mockResolvedValueOnce([mockPlan]);
+      jest.spyOn(stripeClient, 'plansRetrieve').mockResolvedValue(mockPlan);
 
-      const result = await manager.getPlan(mockPlan.id);
-      expect(result).toEqual([mockPlan]);
-    });
-
-    it('should throw error if no plan exists', async () => {
-      const mockPlan = StripePlanFactory();
-
-      mockClient.plansRetrieve = jest.fn().mockResolvedValueOnce(undefined);
-
-      expect.assertions(1);
-      expect(manager.getPlan(mockPlan.id)).rejects.toBeInstanceOf(
-        PlanNotFoundError
-      );
```

Comment:
We don't need this test anymore?

Reply:
The underlying code was written incorrectly, as if plansRetrieve _can_ return `undefined` - it can't. It either returns the plan, or it throws an error.

Reply:
I had to remove this code, since now the test can't be written without a typecast (a hint that the underlying code is incorrect!).

I did create a ticket to remove the check from the code itself here: https://mozilla-hub.atlassian.net/browse/FXA-9620